ezgal/ezgal.py

```diff
@@ -1025,7 +1025,7 @@ def get_solar_observed_mags(self,
         mags = np.zeros((len(zs), len(filters)))
 
         # loop through filters one at a time.
-        for (i, filter) in enumerate(filters):
+        for (i, filter) in enumerate([filters]):
```

Comment:
`filters` should already be a list, so wrapping it in brackets is only going to completely change the meaning of this for loop.  What exactly is the problem you are trying to address with this change?

Reply:
yeah i think this problem was solved with the type(str('')) change. Since if instances were not working, filters reach this line as a string. Since I was too paranoid I just upload all the modifications (i.e. this is not raising an error to me probably because not using that definition or if instance).
Sorry for ambiguity, rookie github user here.